opennlp-docs/src/docbkx/corpora.xml

```diff
@@ -144,12 +144,12 @@ F-Measure: 0.9230575441395671]]>
 		<title>Getting the data</title>
 		<para>The data consists of three files per language: one training file and two test files testa and testb.
 		The first test file will be used in the development phase for finding good parameters for the learning system.
-		The second test file will be used for the final evaluation. Currently there are data files available for two languages:
+		The second test file will be used for the final evaluation. Currently, there are data files available for two languages:
 		Spanish and Dutch.
 		</para>
 		<para>
 		The Spanish data is a collection of news wire articles made available by the Spanish EFE News Agency. The articles are
-		from May 2000. The annotation was carried out by the <ulink url="http://www.talp.cat/">TALP Research Center</ulink> of the Technical University of Catalonia (UPC)
+		from May 2000. The annotation was carried out by the <ulink url="https://www.talp.cat/">TALP Research Center</ulink> of the Technical University of Catalonia (UPC)
 		and the <ulink url="http://clic.ub.edu/">Center of Language and Computation (CLiC)</ulink>of the University of Barcelona (UB), and funded by the European Commission
```

Comment:
http://clic.ub.edu/ is down for me.

Reply:
Any alternative?

Reply:
Not really. Maybe: [this](https://www.ub.edu/portal/web/filologia-comunicacio/centre-de-llenguatge-i-computacio-clic-) - last internet archive is from March 2022 (seems to be down a while now): http://web.archive.org/web/20220516042208/http://clic.ub.edu/